vxfreeswitch/tests/test_voice.py

```diff
@@ -549,6 +550,30 @@ def create_worker(self, config={}):
         return self.tx_helper.get_transport(default)
 
     @inlineCallbacks
+    def wait_for_client_registration(self, worker, clientid):
+        '''Waits until the client has registered itself to the worker. Returns
+        False if it times out waiting.'''
+        for _ in range(5):
+            if worker._originated_calls.get(clientid, None) is None:
+                returnValue(True)
+            d = defer.Deferred()
+            reactor.callLater(0.01, d.callback, None)
+            yield d
```

Comment:
I'm wondering if we should make a tiny `sleep` helper method that does:
```python
SLEEP_TIME = 0.01

def sleep(self, time=SLEEP_TIME):
    d = defer.Deferred()
    reactor.callLater(time, d.callback, None)
    return d
```
and then just do `yield self.sleep()` in the two tests? Mostly I just want us to easily be able to tweak the sleep constant.